orangecanvas/document/quickmenu.py

```diff
@@ -1454,18 +1681,26 @@ def eventFilter(self, obj, event):
         etype = event.type()
         if etype == QEvent.KeyPress:
             key = event.key()
+            # down
             if key == Qt.Key_Down:
                 self.moveCurrent(1, 0)
                 return True
+            # up
             elif key == Qt.Key_Up:
                 self.moveCurrent(-1, 0)
                 return True
-            elif key == Qt.Key_Tab:
-                self.moveCurrent(0, 1)
-                return  True
+            # enter / return
             elif key == Qt.Key_Enter or key == Qt.Key_Return:
                 self.activateCurrent()
                 return True
+            # shift + tab / left
+            elif key == Qt.Key_Backtab or key == Qt.Key_Left:
+                self.parent().previousPage()
+                return True
+            # tab / right
+            elif key == Qt.Key_Tab or key == Qt.Key_Right:
+                self.parent().nextPage()
+                return True
```

Comment:
This Left/Right key filter interferes with the search text line edit cursor navigation.